src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -276,14 +276,21 @@ TEST(MessageHelperTestCreate,
 
   smart_objects::SmartObject& object = *smartObjectPtr;
 
+  uint64_t app_id = 1;
+  std::string cmd_icon_value = "10";
+  int64_t cmd_id = 5;
+  uint32_t internal_id = 1;
```

Comment:
@APipko Why these values are non-const?

Reply:
@AByzhynar fixed in 360382e6f413851ab2931500eee7f46130872c7b